solaris/utils/io.py

```diff
@@ -1,6 +1,6 @@
 """Utility functions for data io."""
 import numpy as np
-import skimage
+import skimage.io
```

Comment:
Can you clarify why you did the `skimage.io` imports instead of just `skimage` in a few places? Thanks!

Reply:
It's an issue with skimage versions above 16, I was getting errors saying that `skimage has no module io`. I was testing after installing solaris with pip into an existing environment, forgot to make a note of this. Later I switched to testing with a fresh solaris environment set up with pip following the README instructions.

Reply:
Weird. Thanks for clarifying!